app/helpers/profiles_helper.rb

```diff
@@ -1,2 +1,8 @@
 module ProfilesHelper
+  def favorite_check(favorite_profile, current_user)
+    profile = Profile.find_by(user_id: current_user.id)
+    relation = Favorite.where(profile_id: favorite_profile.id, favorite_profile_id: profile.id).first
+    return false if relation.nil?
```

Comment:
I'd separate return statement with empty lines

Reply:
Of course, I forgot about it